coderd/database/generate.sh

```diff
@@ -13,8 +13,10 @@ SCRIPT_DIR=$(dirname "${BASH_SOURCE[0]}")
 (
 	cd "$SCRIPT_DIR"
 
-	# Dump the updated schema.
-	go run gen/dump/main.go
+	echo generate 1>&2
+
+	# Dump the updated schema (use make to utilize caching).
+	make -C ../.. --no-print-directory coderd/database/dump.sql
```

Comment:
Technically make would still think the old `go run gen/dump/main.go` method was new and wouldn't make you do it again, since make doesn't actually have a database (make determines that a target is outdated if any of it's dependencies are newer mtime than the target itself).

I do prefer calling make though so 👍